static/src/javascripts/projects/commercial/modules/prebid/utils.spec.js

```diff
@@ -358,14 +358,30 @@ describe('Utils', () => {
         }
     });
 
-    test('shouldIncludeMobileSticky should be true if location cookie is NA, switch is ON and content is Article on mobiles ', () => {
+    test('shouldIncludeMobileSticky should be true if geolocation is NA, switch is ON and content is Article on mobiles ', () => {
```

Comment:
This is minor, but there's a bit of repetition on these tests, you could do something like

```js
['US', 'AU', 'NZ'].forEach(region => {
    test(`should include mobile sticky if geolocation is ${region}, switch is ON and content is Article on mobiles`, () => {
        config.set('page.contentType', 'Article');
        config.set('switches.mobileStickyLeaderboard', true);
        getSync.mockReturnValue(region);
        isBreakpoint.mockReturnValue(true);
        expect(shouldIncludeMobileSticky()).toBe(true);
    });
});
```